gfauto/gfauto/amber_converter.py

```diff
@@ -633,7 +639,7 @@ def spirv_asm_shader_job_to_amber_script(
 ) -> Path:
 
     log(
-        f"Amberfy: {str(shader_job_file_amber_test.variant_asm_spirv_job.asm_spirv_shader_job_json)} "
+        f"Amberfy: {str(variant.asm_spirv_shader_job_json) for variant in shader_job_file_amber_test.variants_asm_spirv_job} "
```

Comment:
My IDE doesn't like this, but it works! Unfortunately, I think it prints "<generator object spirv_asm_shader_job_to_amber_script.<locals>.<genexpr> at 0x7fb51f4caaf0>" so please change it anyway! Wrapping it in `[` `]` seems to work for me.